### PR TITLE
got it working

### PR DESCRIPTION
**Pull request by jay-anderson**

### CODE REVIEW

File: client-side/src/pages/DataForm.jsx [hunk 1]
```diff
@@ -56,27 +56,33 @@ export default class DataForm extends Component {
...
     handleSubmit = async (e) => {
         e.preventDefault();
-        this.props.history.push('/predict') //redirects to predict route
-
 
         // test body form to trigger backend response
-        var bodyFormData = new FormData();
-        bodyFormData.append('description', 'example description');
-        bodyFormData.append('imageData', 'example image stub');
+        //var bodyFormData = new FormData();
```
Comment (aaa1008) [inline client-side/src/pages/DataForm.jsx:61]: I'd remove the unused comments

File: client-side/src/pages/DataForm.jsx [hunk 2]
```diff
@@ -56,27 +56,33 @@ export default class DataForm extends Component {
...
+            this.state.size+ '+' + this.state.image_url;
 
-        await axios.post('http://127.0.0.1:5000/predict', bodyFormData)
+        await axios.get('http://127.0.0.1:5000/predict?seq=' + bodyFormData)
             .then((result) => {
-                this.setState({
-                    isLoaded: true,
-                    data: result,
-                })
+                //console.log(result);
```
Comment (aaa1008) [inline client-side/src/pages/DataForm.jsx:69]: more comments to remove

File: client-side/src/pages/PredictionPage.jsx [hunk 3]
```diff
@@ -1,20 +1,37 @@
...
     constructor(props) {
         super(props)
-        this.state = {
-            categories: [
-                {category: "jeans", prediction: "95%"},
-                {category: "shirt", prediction: "5%"}
-            ]
-        }
-        }
+        //this.categories = this.props.location.data.data;
```
Comment (aaa1008) [inline client-side/src/pages/PredictionPage.jsx:8]: more comments to remove

File: client-side/src/pages/PredictionPage.jsx [hunk 4]
```diff
@@ -1,20 +1,37 @@
...
+            //this.state.data = this.props.location;
+            //const { data } = this.props.location
+            this.state = {
+                //add global variable here -- not thread safe
+                categories: this.props.location.data.data
+            };
+
+            console.log(this.state.categories);
```
Comment (aaa1008) [inline client-side/src/pages/PredictionPage.jsx:16]: remove logs

File: client-side/src/pages/PredictionPage.jsx [hunk 5]
```diff
@@ -26,7 +43,8 @@ class PredictionPage extends Component {
     }
 
     renderTableHeader() {
-        let header = Object.keys(this.state.categories[0])
+        //console.log(Object.keys(this.state.categories))
```
Comment (aaa1008) [inline client-side/src/pages/PredictionPage.jsx:46]: commented log to remove

**Review by aaa1008 — APPROVED** [review]
Approving for the sake of time/how much we worked on it last night. We'd ideally clean-up the logs/comments.

**Review by sms1097 — APPROVED** [review]
LGTM

Approving for time sake, we need to go back and clean the entire project up for styling anyways, so might as well just run this through with logs

**Pull request merged**